includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -253,28 +255,30 @@ public function process_subscription_payment( $amount, $renewal_order, $retry =
 			if ( isset( $_REQUEST['process_early_renewal'] ) ) { // wpcs: csrf ok.
```

Comment:
This comment `wpcs: csrf ok.` isn't necessary or should be replaced by something like `phpcs:ignore ...` instead.